stream-platform/src/main/java/io/camunda/zeebe/stream/api/scheduling/ScheduledCommandCache.java

```diff
@@ -48,6 +50,9 @@ public boolean contains(final Intent intent, final long key) {
     @Override
     public void remove(final Intent intent, final long key) {}
 
+    @Override
+    public void clear() {}
```

Comment:
🔧 Not sure whether you want to backport the last commit from the other PR here as well